recipes/pytest-notification/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pytest-notification" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e6939b1ae54b7f13721516a37204db9a1ccf278b91c5e0d8491f00ab73be7bcc
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - pytest >=4
+    - python >=3.5
+    - setuptools
+  run:
+    - pytest >=4
+    - python >=3.5
+    - setuptools
```

Comment:
Just to double-check, do you really need ``setuptools`` to run this project?

Reply:
@marcelotrevisani Thanks for noticing! This was added by conda skeleton while it is not listed in the requirements. I removed it. Thanks!